src/dockerfiles/Makefile

```diff
@@ -0,0 +1,60 @@
+.DEFAULT_GOAL = all
+
+# By default, turn off the Makefile practice of printing each command before
+# you run it.
+ifndef VERBOSE
+.SILENT:
+endif
+
+build-function:
+	docker build . -t aqueducthq/function -f function/function.dockerfile --no-cache
+
+build-function-test:
```

Comment:
@hsubbaraj-spiral You should be able to run this make command and `make build-snowflake-test` from the `src/dockerfiles` directory